packages/glimmer-runtime/lib/dom/attribute-managers.ts

```diff
@@ -110,11 +110,16 @@ export class PropertyManager extends AttributeManager {
   }
 
   updateAttribute(env: Environment, element: Element, value: Opaque, namespace?: DOMNamespace) {
-    if (isAttrRemovalValue(value)) {
-      this.removeAttribute(env, element, namespace);
-    } else {
-      this.setAttribute(env, element, value, namespace);
+    let attr = this.attr;
+    if (value === undefined) {
+      value = null;
+    }
+
+    if (value === null && (attr === 'value' || attr === 'type' || attr === 'src')) {
```

Comment:
This kinda sucks 😢 

I know that this was what we were doing in HTMLBars, but should we at least tie the `attr` and `tagName` together?